### PR TITLE
Add affirmation overrides config option and add default xenon ones ne…

### PR DESCRIPTION
**Pull request by jferrant**
Closes: https://github.com/stacks-network/stacks-core/issues/4493

### CODE REVIEW

File: testnet/stacks-node/src/config.rs [hunk 1]
```diff
@@ -1302,9 +1382,34 @@ pub struct BurnchainConfigFile {
     pub sunset_end: Option<u32>,
     pub wallet_name: Option<String>,
     pub ast_precheck_size_height: Option<u64>,
+    pub affirmation_overrides: Option<Vec<AffirmationOverride>>,
 }
 
 impl BurnchainConfigFile {
+    /// Add affirmation overrides required to sync Xenon
```
Comment (jcnelson) [inline testnet/stacks-node/src/config.rs:1389]: Maybe add a comment to explain to the reader how these affirmation maps can be independently recreated?

**Review by jcnelson — DISMISSED** [review]
LGTM; just one minor nit

File: testnet/stacks-node/src/config.rs (hunk 1 above)
Comment (jferrant) [reply testnet/stacks-node/src/config.rs:1389]: Good idea. I have no idea where these came from XD I just got them off @wileyj .

Comment (jferrant) [reply testnet/stacks-node/src/config.rs:1389]: @kantai do you have any insights?

**Review by wileyj — APPROVED** [review]
shipit!

**Review by jcnelson — DISMISSED** [review]

**Review by kantai — APPROVED** [review]

**Review by wileyj — APPROVED** [review]

**Pull request merged**